Paez

pecasdesign opened the pull request:
Link

a0viedo committed:
Merge pull request #2 from a0viedo/Sponsor-Ciudad

Ciudad Inteligente

a0viedo committed:
Merge pull request #1 from a0viedo/BOTON_SPONSOR

botón sponsor!

Commit:
Merge remote-tracking branch 'bevacqua/gh-pages' into gh-pages

Commit:
Merge remote-tracking branch 'bevacqua/gh-pages' into gh-pages

Commit:
Sharon

Commit:
Merge remote-tracking branch 'bevacqua/gh-pages' into gh-pages

Commit:
Bynens

Commit:
Merge remote-tracking branch 'bevacqua/gh-pages' into gh-pages

Commit:
Buy Tickets!

Commit:
Merge remote-tracking branch 'bevacqua/gh-pages' into gh-pages

Commit:
MuleSoft

Commit:
Merge remote-tracking branch 'bevacqua/gh-pages' into gh-pages

Commit:
Workshop Close

Commit:
Merge remote-tracking branch 'bevacqua/gh-pages' into gh-pages

Commit:
Chau Ashley

Commit:
Merge remote-tracking branch 'bevacqua/gh-pages' into gh-pages

Commit:
Erich

Commit:
Matthew

a0viedo committed:
Merge remote-tracking branch 'upstream/gh-pages' into gh-pages

a0viedo committed:
Merge branch 'matthew' into gh-pages

Commit:
Merge remote-tracking branch 'bevacqua/gh-pages' into gh-pages

Commit:
Pino

Commit:
Merge remote-tracking branch 'origin/gh-pages' into gh-pages

Commit:
Merge remote-tracking branch 'bevacqua/gh-pages' into gh-pages

Commit:
Baez

Commit:
Merge remote-tracking branch 'bevacqua/gh-pages' into gh-pages

Commit:
Paez

bevacqua commented on paez.html:
```diff
@@ -0,0 +1,99 @@
+        <div class="icon icon-dots tres glitch" data-text="&#xe903;">&#xe903;</div>
+        <div class="col-md-6"><div class="profile-img"><img src="images/paez.jpg"></div><div class="profile-img-trama"></div></div>
+        <div class="col-md-6"><div class="profile-text"><h2>Diego Paez</h2><a href="https://twitter.com/carax" target="_blank">@carax</a>
+          <p>Diego is a JS developer with a degree in Computer Science and a passion for HCI. He was born in the *southernmost* place in America, **Tierra del Fuego**. He moved to La Plata where he got his degree at UNLP. Diego has co-founded LaPlataJS a local JS community and [GEUT](http://geutstudio.com/) a *mysterious* side project. He is currently working for Despegar, the biggest web travel  ...
```
Esto debería ser el HTML compilado:

```
<div><p>Diego is a JS developer with a degree in Computer Science and a passion for HCI. He was born in the <em>southernmost</em> place in America, <strong>Tierra del Fuego</strong>. He moved to La Plata where he got his degree at UNLP. Diego has co-founded LaPlataJS a local JS community and <a href="http://geutstudio.com/">GEUT</a> a <em>mysterious</em> side project. He is currently working for Despegar, the biggest web travel agency in Latin America.</p></div>
```

Commit:
em-strong

Esta tipografía no tiene esas variantes…
por ende mas q cambiarle apenas el tono no se puede.

bevacqua commented:
Gracias :tada:
